server/routes/userRoutes.js

```diff
@@ -1,70 +1,136 @@
 import express from 'express';
 import dotenv from 'dotenv';
 import pkg from 'pg';
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import verifyToken from '../middleware/jwtMiddleware.js'
 
 dotenv.config();
 const router = express.Router();
 const { Pool } = pkg;
 const db = new Pool({
-    connectionString: process.env.DATABASE_URI
+  connectionString: process.env.DATABASE_URI
 });
```

Comment:
Nit: It might be cleaner to move the database connection logic into its own file or folder. This would help with organization and make it easier to manage and reuse in the future.